db/schema_applier.hh

```diff
@@ -22,6 +23,68 @@ namespace schema_tables {
 
 future<> merge_schema(sharded<db::system_keyspace>& sys_ks, distributed<service::storage_proxy>& proxy, gms::feature_service& feat, std::vector<mutation> mutations, bool reload = false);
 
+enum class table_kind { table, view };
+
+struct table_selector {
+    bool all_in_keyspace = false; // If true, selects all existing tables in a keyspace plus what's in "tables";
+    std::unordered_map<table_kind, std::unordered_set<sstring>> tables;
```

Comment:
Why not 
``` C++
enum class table_kind { table = 0, view };
... 
std::array<std::unordered_set<sstring>> tables, 2> tables;
```

Reply:
This is not performance critical, so saving brain power (less tricks) wins.

Reply:
Ok.